main.go

```diff
@@ -149,6 +149,8 @@ func parseConfiguration(logger *logrus.Logger) (ServiceConfig, error) {
 	}
 	if jwks := os.Getenv(envAccountsJWKSFile); jwks != "" {
 		config.JWKSFile = jwks
+	} else {
+		config.JWKSFile = jwt.AccountsJWKSFile
```

Comment:
Accidentally discovered a minor bug: If the env var that defines the JWKS file to use is not set we didn't fall back to the default value. Now we do.